pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TopicTransactionBufferRecoverTest.java

```diff
@@ -486,7 +486,7 @@ public void testTransactionBufferRecoverThrowPulsarClientException() throws Exce
         TransactionBufferSnapshotService transactionBufferSnapshotServiceOriginal =
                 (TransactionBufferSnapshotService) field.get(getPulsarServiceList().get(0));
         // mock reader can't read snapshot fail
-        doThrow(new PulsarClientException("test")).when(reader).hasMoreEvents();
+        doThrow(new RuntimeException("test")).when(reader).hasMoreEvents();
```

Comment:
```suggestion
        doThrow(new Exception("test")).when(reader).hasMoreEvents();
```

PulsarClientException is not a RuntimeException, If use RuntimeException here, we will lose the previous test coverage.